prow/gerrit/adapter/trigger.go

```diff
@@ -36,15 +40,35 @@ func messageFilter(lastUpdate time.Time, change client.ChangeInfo, presubmits []
 			continue
 		}
 
-		if !pjutil.TestAllRe.MatchString(message.Message) {
+		if pjutil.TestAllRe.MatchString(message.Message) {
+			filters = append(filters, pjutil.TestAllFilter())
+		} else if pjutil.RetestRe.MatchString(message.Message) {
```

Comment:
In the GitHub world this is an `if` not an `else if`

Reply:
else will save us one comparison if test all matches :)

Reply:
That's not my point -- in GitHub you can do `/retest` _and_ `/test non-required-job` in one comment and we will do both. Your impl is not DRY with the GitHub approach and is affording a different UX

Reply:
ahh, makes sense. Done.